source/get-started/download-and-install.txt

```diff
@@ -0,0 +1,102 @@
+.. _php-download-and-install:
+
+====================
+Download and Install
+====================
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: setup, composer, installation, code example
+
+.. procedure::
+   :style: connected
+
+   .. step:: Install dependencies
+
+      Before you begin developing, ensure that you have the following
+      dependencies installed on your local machine:
+      
+      - `PHP <https://www.php.net/manual/en/install.php>`__ version 7.4 or later
+      - `Composer <https://getcomposer.org/download/>`__ version 2.0 or later
+    
+   .. step:: Install the MongoDB PHP extension
+
+      Run the following command to install the ``mongodb`` PHP extension:
+      
+      .. code-block:: bash
+
+         sudo pecl install mongodb
+
+   .. step:: Update your PHP configuration file
+
+      To enable the ``mongodb`` extension in your PHP configuration file, add the
+      following line to the top of your ``php.ini`` file:
```

Comment:
Q: Does the php.ini file exist already once you download the dependency? Where would it be installed?

Reply:
Yes, when you install PHP. You can see where it's installed with php --ini command (the admonition below) - for me, I used homebrew to install, so it's in /opt/homebrew/etc/php/8.3